### ISSUE TITLE
terminate called after throwing an instance of 'std::invalid_argument'

### ISSUE THREAD

**Issue by tseemann** [andersen-lab/ivar#7]
using 1.1 and htslib 1.10.2

```
% ivar trim -i ivar.bam -b nCoV-2019.scheme.bed -p out

terminate called after throwing an instance of 'std::invalid_argument'
 what():  stoi
Aborted (core dumped)
```
Maybe it's the BED:
```
% cut -f1-3 nCoV-2019.scheme.bed > new.bed

ivar trim -i ivar.bam -b nww.bed -p out
Number of references: 1
Reference Name: MN908947
Reference Length: 29903
Using Region: MN908947
Sorted By Coordinate
Sorted By Query Name
Segmentation fault (core dumped)
```
Any ideas?

**Comment by tseemann** [comment]
via @drpowell
```
I think the reason it does on your original bed is because a score is expected in col 5 which you don't have
https://github.com/andersen-lab/ivar/blob/master/src/primer_bed.cpp#L88
(and a strand in col 6)
```

**Comment by tseemann** [comment]
See also https://github.com/artic-network/artic-ncov2019/issues/12

**Comment by gkarthik** [comment]
I think that is the source of the error. I use the BED format from here -  https://genome.ucsc.edu/FAQ/FAQformat.html#format1. 

The order I followed is,
```
chrom, chromStart,chromEnd, name, score, strand
```
It seems to match the default 6 column BED format generated by `bedtools bamtobed`. I assumed that order was standard (maybe I'm in the wrong there?). 

This issue does bring up the lack of checks while reading the BED file. I'll get on it :).

**Comment by tseemann** [comment]
Yes it seems @scalene has produced invalid BED files.
Some robustness in `ivar` would be appreciated however :-)

**Comment by gkarthik** [comment]
Closed #13
